content/tracing/languages/java.md

```diff
@@ -18,7 +18,7 @@ further_reading:
 
 ## Installation and Getting Started
 
-To begin tracing applications written in any language, first [install and configure the Datadog Agent][3] (see additional documentation for [tracing Docker applications](/tracing/setup/docker/)).
+To begin tracing applications written in any language, first [install and configure the Datadog Agent][3] (see additional documentation for [tracing Docker applications](//agent/docker/apm/)).
```

Comment:
Why does it link to `//agent/docker/apm/` instead of `/agent/docker/apm/`?

Reply:
wrong link :)